pkg/agent/agentserver/tunnel.go

```diff
@@ -121,7 +121,11 @@ func (t *Tunnel) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 				},
 			},
 		}
-		t.Server.Backend.Send(packet)
+		err = t.Server.Backend.Send(packet)
+		if err != nil {
+			klog.Errorf("Error sending packet %v", err)
```

Comment:
Sorry should have caught this before. Errors should start with a lower case letter.

Reply:
Fixed